src/OSPSuite.Core/Domain/Formulas/TableFormula.cs

```diff
@@ -279,14 +276,15 @@ public override void UpdatePropertiesFrom(IUpdatable source, ICloneManager clone
 
          var tableFormula = source as TableFormula;
          if (tableFormula == null) return;
-         ClearPoints();
-         tableFormula.AllPoints().Each(p => AddPoint(p.Clone()));
          XDimension = tableFormula.XDimension;
          XName = tableFormula.XName;
          YName = tableFormula.YName;
          UseDerivedValues = tableFormula.UseDerivedValues;
          _xDisplayUnit = tableFormula._xDisplayUnit;
          _yDisplayUnit = tableFormula._yDisplayUnit;
+
+         ClearPoints();
```

Comment:
We could try to optimize and not throw changed all the time but I think it does not make sense.
Theoreticlally we should also do it when a single value is being changed, but it's much harder and we don't use it that way anywyas (always clone of the whole table)